packages/test/test-end-to-end-tests/src/test/gc/gcContainerRuntimeExpiry.spec.ts

```diff
@@ -101,9 +96,9 @@ import { mockConfigProvider } from "./mockConfigProivder";
         clock.tick(1);
         assert(container1.closed, `Container1 should be closed, it has should have reached its session expiry timeout.
             Current: ${clock.now}, expected: ${clockEnd}`);
-    });
+    }).timeout(10000);
```

Comment:
We shouldn't use extended timeout unless necessary. For these tests, we should add unit tests and remove the end-to-end tests as this is just testing functionality confined to the container runtime / GC layer.